<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-JAL21INA" failures="2" tests="2" name="Default test(failed)" time="0.102" errors="0" timestamp="2023-01-21T16:17:41 IST">
  <testcase classname="com.BasicTestNG.AssertionsIntestNg" name="logout" time="0.025">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at com.BasicTestNG.AssertionsIntestNg.logout(AssertionsIntestNg.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logout -->
  <testcase classname="com.BasicTestNG.AssertionsIntestNg" name="test1" time="0.009">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true],
	expected [false] but found [true],
	expected [false] but found [true],
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true],
expected [false] but found [true],
expected [false] but found [true],
expected [false] but found [true]
at com.BasicTestNG.AssertionsIntestNg.test1(AssertionsIntestNg.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
